test/visit.ts

```diff
@@ -26,9 +26,7 @@ describe("types.visit", function() {
         assert.ok(thisExp.test(withThis));
 
         types.visit(ast, {
-            visitThisExpression: function() {
-                return builders.identifier("self");
-            }
+            visitThisExpression: () => builders.identifier("self")
```

Comment:
Hmm. These probably shouldn't be arrow functions in case someone wants to edit them to reference `this`.